ttnn/cpp/ttnn/operations/index_fill/index_fill_pybind.cpp

```diff
@@ -0,0 +1,44 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "index_fill_pybind.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "index_fill.hpp"
+#include "ttnn/cpp/pybind11/decorators.hpp"
+#include "ttnn/operations/index_fill/device/index_fill_device_operation.hpp"
+
+namespace py = pybind11;
+
+namespace ttnn::operations::index_fill {
+
+void bind_index_fill_operation(py::module& module) {
```

Comment:
maybe out of the scope of this but can we look at in the future supporting the square bracket notation on the left hand side.

e.g x[:2] = y

is this something we should do at a higher level @ayerofieiev-tt ?